src/views/Dashboard.vue

```diff
@@ -31,8 +31,7 @@
 			<DashboardWidgetItem :target-url="getItemTargetUrl(item)"
 				:main-text="getMainText(item)"
 				:sub-text="getSubText(item)"
-				:item="item"
-				v-on="handlers">
```

Comment:
was it maybe meant to be `v-on="$listeners"`?

Reply:
No, I copied the content from:
https://github.com/nextcloud/nextcloud-vue-dashboard/blob/de7e24fdbb0f53cdbe0affddfd219bbd335399a2/src/DashboardWidget.vue#L46
And while adjusting forgot to remove it. We have no handlers in talk